addons/l10n_br_website_sale/views/templates.xml

```diff
@@ -28,4 +28,16 @@
             </t>
         </xpath>
     </template>
+
+    <template id="total" inherit_id="website_sale.total">
+        <tr id="order_total_untaxed" position="attributes">
+            <attribute name="t-att-style">'display: none;' if website.company_id.country_code == "BR" else ''</attribute>
+        </tr>
+        <tr id="order_total_taxes" position="attributes">
+            <attribute name="t-att-style">'display: none;' if website.company_id.country_code == "BR" else ''</attribute>
+        </tr>
+        <tr id="order_total" position="attributes">
+            <attribute name="t-att-class">'border-top' if website.company_id.country_code != "BR" else ''</attribute>
+        </tr>
+    </template>
```

Comment:
The essence is good, but you should generally use add="..." when dealing with the attributes (you could be replacing this attribute with the empty string `''` if it were used anywhere else).

```suggestion
        <tr id="order_total_untaxed" position="attributes">
            <attribute name="t-attf-class" add="#{'d-none' if website.company_id.country_code == 'BR' else ''}" separator=" "/>
        </tr>
        <tr id="order_total_taxes" position="attributes">
            <attribute name="t-attf-class" add="#{'d-none' if website.company_id.country_code == 'BR' else ''}" separator=" "/>
        </tr>
        <tr id="order_total" position="attributes">
            <attribute name="t-attf-class" add="#{'border-top' if website.company_id.country_code != 'BR' else ''}" separator=" "/>
        </tr>

```